psi4/share/psi4/databases/O24by5.py

```diff
@@ -0,0 +1,2628 @@
+""")
+
+GEOS["%s-%s-dimer" % (dbse, "23-1.0")] = qcdb.Molecule("""
+0   3
+He       -1.931500000     0.000000000     0.000000000
+--
+0   3
+He        1.931500000     0.000000000     0.000000000
+
+units angstrom
+no_reorient
+no_com
+symmetry c1
+""")
+
+GEOS["%s-%s-dimer" % (dbse, "23-1.2")] = qcdb.Molecule("""
+0   3
+He       -2.317800000     0.000000000     0.000000000
+--
+0   3
+He        2.317800000     0.000000000     0.000000000
+
+units angstrom
+no_reorient
+no_com
+symmetry c1
+""")
+
+GEOS["%s-%s-dimer" % (dbse, "23-1.5")] = qcdb.Molecule("""
+0   3
+He       -2.897250000     0.000000000     0.000000000
+--
+0   3
+He        2.897250000     0.000000000     0.000000000
+
+units angstrom
+no_reorient
+no_com
+symmetry c1
+""")
+
+GEOS["%s-%s-dimer" % (dbse, "23-2.0")] = qcdb.Molecule("""
+0   3
+He       -3.863000000     0.000000000     0.000000000
+--
+0   3
+He        3.863000000     0.000000000     0.000000000
+
+units angstrom
+no_reorient
+no_com
+symmetry c1
+""")
+
+GEOS["%s-%s-dimer" % (dbse, "24-0.9")] = qcdb.Molecule("""
+1   2
+C         0.000000000    -0.436434806     0.497570949
+O         0.000000000     0.196238679     1.420138459
+--
+0   1
+F         0.000000000     0.109938945    -0.992070159
+H         0.000000000    -0.582542676    -1.636605903
+
+units angstrom
+no_reorient
+no_com
+symmetry c1
+""")
+
+GEOS["%s-%s-dimer" % (dbse, "24-1.0")] = qcdb.Molecule("""
+1   2
+C         0.000000000    -0.444773807     0.611409007
+O         0.000000000     0.187899678     1.533976518
+--
+0   1
+F         0.000000000     0.118277946    -1.105908217
+H         0.000000000    -0.574203674    -1.750443962
+
+units angstrom
+no_reorient
+no_com
+symmetry c1
+""")
+
+GEOS["%s-%s-dimer" % (dbse, "24-1.2")] = qcdb.Molecule("""
+1   2
+C         0.000000000    -0.461451810     0.839085125
+O         0.000000000     0.171221675     1.761652635
+--
+0   1
+F         0.000000000     0.134955949    -1.333584335
+H         0.000000000    -0.557525671    -1.978120079
+
+units angstrom
+no_reorient
+no_com
+symmetry c1
+""")
+
+GEOS["%s-%s-dimer" % (dbse, "24-1.5")] = qcdb.Molecule("""
+1   2
+C         0.000000000    -0.486468814     1.180599300
+O         0.000000000     0.146204671     2.103166811
+--
+0   1
+F         0.000000000     0.159972954    -1.675098511
+H         0.000000000    -0.532508667    -2.319634255
+
+units angstrom
+no_reorient
+no_com
+symmetry c1
+""")
+
+GEOS["%s-%s-dimer" % (dbse, "24-2.0")] = qcdb.Molecule("""
+1   2
+C         0.000000000    -0.528163822     1.749789594
+O         0.000000000     0.104509663     2.672357104
+--
+0   1
+F         0.000000000     0.201667961    -2.244288804
+H         0.000000000    -0.490813659    -2.888824548
+
+units angstrom
+no_reorient
+no_com
+symmetry c1
+""")
+
+# <<< Derived Geometry Strings >>>
+for rxn in HRXN:
+    GEOS["%s-%s-monoA-CP" % (dbse, rxn)] = GEOS["%s-%s-dimer" % (dbse, rxn)].extract_fragments(
+        1,
+        [
+            2,
+        ],
+    )
```

Comment:
is there a reason to not allow the non-CP case for these databases, like https://github.com/psi4/psi4/blob/master/psi4/share/psi4/databases/S22.py#L879-L880  and in ACTV?

Reply:
enabled it, but not sure how it works (if at all) with `mb` variant.